docs/next/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "build-master": "VERSIONING_DISABLED=true next build",
     "start": "next start",
-    "test": "jest",
+    "test": "jest --transformIgnorePatterns 'node_modules/(?!react-github-btn)/'",
```

Comment:
Can this be done in a jest config file instead?

Reply:
hmm i moved it to jest config as such:
```js

const customJestConfig = {
  moduleDirectories: ['node_modules', '<rootDir>/'],
  testEnvironment: 'node',
  transformIgnorePatterns: ['node_modules/(?!react-github-btn)'], // here
};
```

but got this error: https://gist.github.com/yuhan/f54b1069a76adf515d4eec02604fae12

Reply:
Ah, ok. Maybe we can just mock that module? Not a huge deal and we can figure it out it in a followup.